tests/e2e/e2e_test.go

```diff
@@ -3492,24 +3460,28 @@ func TestE2E(t *testing.T) {
 						Description: "A test tailored profile to test kubelet remediation",
 						EnableRules: []compv1alpha1.RuleReferenceSpec{
 							{
-								Name:      prefixName(pbName, requiredRule),
+								Name:      "ocp4-kubelet-enable-streaming-connections",
+								Rationale: "To be tested",
+							},
+							{
+								Name:      "ocp4-version-detect-in-ocp",
```

Comment:
We're not checking the status of this below anywhere. Do we need it, or should we add an assertion for it?

Reply:
do you mean the checkResult, I think it is being check at the end, on line 3565,
```
				err = assertHasCheck(f, suiteName, scanName, checkResult)
```